### PR TITLE
Drawer: added height property for expanded state

### PR DESCRIPTION
**Pull request by ianmcburnie**
The drawer should always grow to 95% height when expanded fully, so I added a quick fix for that.

I also condensed the two examples just down to a single example and reworded things slightly.

I also added storybook stories to capture the other example and edge cases we need to test.

Screen Shot 2020-04-21 at 9 02 41 PM

### CODE REVIEW

File: src/less/drawer/base/drawer.less [hunk 1]
```diff
@@ -68,6 +68,7 @@
 }
 
 .drawer__window--expanded {
+    height: 95%;
     max-height: 95%;
```
Comment (agliga) [inline src/less/drawer/base/drawer.less:72]: I'm fine with it, but what is the point of having both `height` and `max-height`

Comment (agliga) [reply src/less/drawer/base/drawer.less:72]: Never mind, I remembered this is to override the other `max-height`

Comment (ianmcburnie) [reply src/less/drawer/base/drawer.less:72]: I believe it prevents an override of `height: 100%`
